src/Node/Info.php

```diff
@@ -221,9 +221,11 @@ public function refresh(array $params = []): Response
         $this->_response = $this->getNode()->getClient()->requestEndpoint($endpoint);
         $data = $this->getResponse()->getData();
 
-        $this->_data = \reset($data['nodes']);
-        $this->_id = \key($data['nodes']);
-        $this->getNode()->setId($this->getId());
+        if (null !== $nodeId = \array_key_first($data['nodes'])) {
```

Comment:
Trying to get my head around on what happened before when we didn't check furl null. 

If there are no nodes, it is expected that we keep the old values or should these be overwritten with empty values?

Reply:
You're right, I didn't think this case, when nodes can be empty